src/doc/en/developer/portability_testing.rst

```diff
@@ -219,7 +219,7 @@ on our container to install the necessary build prerequisites::
   ...
   Done.
 
-(The Sage `Installation Guide <../installation/index.html>`_ also
+(The Sage :ref:`Installation Guide <installation-guide>`_ also
```

Comment:
```suggestion
(The Sage :ref:`Installation Guide <installation-guide>` also
```
no underscores in the :ref: syntax -- please check and fix throughout

Reply:
I have done those changes.